### PR TITLE
Pass initial diskSize to databases, use in provisioning

### PR DESCRIPTION
**Pull request by mixonic**
@bantic for you to review/merge. Closes https://github.com/aptible/diesel.aptible.com/issues/117

The `initialDiskSize` attribute is part of the first-app page. It is saved with a database upon creation, and then used for `diskSize` when provisioning occurs.

The normal database creation path (non-ftux) already handles setting the `diskSize` correctly (on the operation).

### CODE REVIEW

**Comment by bantic** [comment]
@mixonic looks good. I can do the rebase if you'd like and/or are having difficulty. The payment info test changed a lot in my other merged PR

**Comment by mixonic** [comment]
@bantic cleaned up! Read for review.

File: tests/acceptance/welcome/payment-info-test.js [hunk 1]
```diff
@@ -13,14 +13,25 @@ function visitPaymentInfoWithApp(options, userData){
...
   andThen(function(){
-    if (!options) { return clickButton('Skip this step'); }
+    if (!options) {
+      return clickButton('Skip this step');
+    }
 
-    if (options.dbType){ click(`.${options.dbType} a`); }
+    if (options.dbType) {
+      click(`.${options.dbType} a`);
+    }
```
Comment (mixonic) [inline tests/acceptance/welcome/payment-info-test.js]: I changed these back to multi-line to make dropping debuggers in there easier.

File: tests/acceptance/welcome/payment-info-test.js [hunk 2]
```diff
@@ -13,14 +13,25 @@ function visitPaymentInfoWithApp(options, userData){
...
+    if (options.dbInitialDiskSize) {
+      andThen(function(){
+        Ember.$('.slider').trigger('slide', options.dbInitialDiskSize);
```
Comment (bantic) [inline tests/acceptance/welcome/payment-info-test.js]: perhaps we need a `triggerSlider` async test helper

Comment (mixonic) [inline tests/acceptance/welcome/payment-info-test.js]: `triggerSlider` will do. I was pained b/c there is also a `set` trigger. It gave me pause from making a `set` helper.

Will make a `triggerSlider`

File: tests/acceptance/welcome/payment-info-test.js [hunk 3]
```diff
@@ -13,14 +13,25 @@ function visitPaymentInfoWithApp(options, userData){
...
+      });
+    }
     for (var prop in options){
       let dasherized = prop.dasherize();
 
-      // db-type is a button, not a fillIn-able input
-      if (dasherized === 'db-type') { continue; }
+      // non-fillIn-able inputs
+      if ('db-type db-initial-disk-size'.w().indexOf(dasherized) !== -1) {
```
Comment (bantic) [inline tests/acceptance/welcome/payment-info-test.js]: funny that this likely works the same with and without the `w()`. It's more correct this way, though.

**Comment by bantic** [comment]
@mixonic cool, lgtm. Back to you to merge when you add the triggerSlider helper

File: tests/acceptance/welcome/payment-info-test.js (hunk 3 above)
Comment (mixonic) [inline tests/acceptance/welcome/payment-info-test.js]: The other caveat is `indexOf` on a string is ecmascript5, no IE8. Not that we care :fire:

**Pull request merged**